src/app/wallet/address-book/address-book.component.ts

```diff
@@ -16,12 +16,12 @@ export class AddressBookComponent {
   }
 
   openNewAddress(): void {
-    const dialogRef = this.dialog.open(NewAddressModalComponent);
+    const dialogRef = this.dialog.open(NewAddressModalComponent, {panelClass: 'small-model'});
```

Comment:
Is it a model or a modal?

Reply:
Yeah, Typo mistake. it should be a modal >> **done **